docker/ubuntu-16.04-cuda8-cudnn6-all-options/Dockerfile

```diff
@@ -0,0 +1,93 @@
+FROM nvidia/cuda:8.0-cudnn6-devel-ubuntu16.04
+MAINTAINER Aaron Markham <aaronmarkham@fb.com>
+
+# caffe2 install with gpu support
+
+########## REQUIRED DEPENDENCIES ################
+RUN apt-get update
+RUN apt-get install -y --no-install-recommends \
+    python-dev \
+    python-pip \
+    git \
+    build-essential \
+    cmake \
+    libprotobuf-dev \
+    protobuf-compiler \
+    libgoogle-glog-dev
+
+RUN git clone --recursive https://github.com/caffe2/caffe2.git && \
+    pip install --upgrade pip setuptools wheel && \
+    pip install future hypothesis numpy protobuf six
+
+########## OPTIONAL DEPENDENCIES ################
+
+RUN apt-get install -y --no-install-recommends \
+    libatlas-base-dev  \
+    libeigen3-dev \
+    libgflags-dev \
+    libgoogle-glog-dev \
+    libgflags-dev \
+    libgtest-dev \
+    libiomp-dev \
+    libleveldb-dev \
+    liblmdb-dev \
+    libopencv-dev \
+    libopenmpi-dev \
+    libsnappy-dev \
+    openmpi-bin \
+    openmpi-doc \
+    python-dev \
+    python-lmdb \
+    python-matplotlib \
+    python-networkx \
+    python-numpy \
+    python-pydot \
+    python-setuptools \
+    python-scipy \
+    python-skimage \
+    wget
+
+RUN rm -rf /var/lib/apt/lists/*
+
+RUN pip install \
+    flask \
+    graphviz \
+    hypothesis \
+    jupyter \
+    matplotlib \
+    pydot \
+    python-nvd3 \
+    pyyaml \
+    requests \
+    scikit-image \
+    scipy \
+    setuptools \
+    tornado
+
+########## INSTALLATION STEPS ###################
+RUN cd caffe2 && mkdir build && cd build
+WORKDIR /caffe2/build/
+RUN cmake .. \
+    -DCUDA_ARCH_NAME=Manual \
+    -DCUDA_ARCH_BIN="35 52 60 61" \
+    -DCUDA_ARCH_PTX="61" \
+    -DUSE_NNPACK=OFF \
+    -DUSE_ROCKSDB=OFF
+RUN make install
+
+ENV PYTHONPATH /usr/local
+ENV LD_LIBRARY_PATH /usr/local/lib:$LD_LIBRARY_PATH
```

Comment:
I just realized I missed this one. Are you suggesting removing `ENV LD_LIBRARY_PATH /usr/local/lib:$LD_LIBRARY_PATH` and adding `ldconfig`? Would it be better to add it to an existing RUN line?